calcom/cal.com
# chore: prettier pre-commit hook

## What does this PR do?

Uses pre-commit hook for prettier formatting.

The pre-commit hook to trigger prettier formatting is already there but since .husky/_ isnt added to github, the contributor has to manually postinstall script once they clone the repo locally. I have removed the "_" from gitignore which allows husky.sh file to be added to git. Now once the contributor makes a commit it automatically triggers prettier formatting.

Fixes #10334

 Loom Video: https://www.loom.com/share/9d957dae25fa4a2aba28fb733e193bf1?sid=6fbffc9e-c933-4b4c-94c8-c6f4dab79d36

## Requirement/Documentation



- .husky folder should be uploaded to git so that the contributor doesnt has to run postinstall script on their machine.

## Type of change


## How should this be tested?

- add unformatted dummy code to any file.
- stage and commit the changes, the pre-commit should trigger and make the formatting changes.

## Mandatory Tasks


## Checklist



- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes
